hasher-matcher-actioner/scripts/set_threatexchange_source

```diff
@@ -43,12 +43,24 @@ elif [ $1 == "local" ]; then
     # Copy and install from source. Copying is necessary. A link would not work.
     # Dockerbuilds can't follow symlinks outside their context directory.
     printf 'Copying current sources to local-threatexchange.\n'
-    cp -r ../python-threatexchange/ $LOCAL_LIB_LOCATION
+    mkdir -p $LOCAL_LIB_LOCATION
+
+    pushd ../python-threatexchange
+    find . -type f \( \
+        -name "*.py" \
+        -o -name "*.rst" \
+        -o -name "*.md"  \
+        -o -name "*.txt" \
+        \) -exec cp --parents {} ../hasher-matcher-actioner/$LOCAL_LIB_LOCATION \;
+    popd
+
+    rm -r $LOCAL_LIB_LOCATION/threatexchange.egg-info
+
```

Comment:
Should have made this into its own PR. Essentially this reduces the time it takes to copy pytx files for local installation of the threatexchange package by only copying the required files. 

Earlier, because of pyc files and sample files, it was taking far longer.